bin/api-docs/are-readmes-unstaged.js

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node
+
+/**
+ * Node dependencies.
+ */
+const { join } = require( 'path' );
+const chalk = require( 'chalk' );
+const execSync = require( 'child_process' ).execSync;
+
+/**
+ * Local dependencies.
+ */
+const getPackages = require( './packages' );
+
+const getUnstagedFiles = () => execSync( 'git diff --name-only', { encoding: 'utf8' } ).split( '\n' ).filter( ( element ) => '' !== element );
+
+const readmeFiles = getPackages().map( ( [ packageName ] ) => join( 'packages', packageName, 'README.md' ) );
+const unstagedFiles = getUnstagedFiles();
+
+const unstagedReadmes = [];
+unstagedFiles.forEach( ( element ) => {
+	if ( readmeFiles.includes( element ) ) {
+		unstagedReadmes.push( element );
+	}
+} );
+
+let exitCode = 0;
+if ( unstagedReadmes.length > 0 ) {
+	exitCode = 1;
```

Comment:
Seems rather than assigning a local variable with a fallback to the "success" code and then explicitly exiting, we could instead just assign the `process.exitCode` property and let the process exit normally?

```js
process.exitCode = 1;
```

https://nodejs.org/api/process.html#process_process_exitcode

Reply:
Added the suggestion in https://github.com/WordPress/gutenberg/pull/18820/commits/f4aa5ff9bf6ddaae7929e7a1169184dfb31b0382